react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+
+export function StopWatch() {
+  const [seconds, setSeconds] = useState(0);
+  const [play, setPlay] = useState(false);
+  const [pauseID, setPauseID] = useState(null);
+
+  function startTimer() {
+    if (!play) {
+
+      const count = setInterval(() => {
+        setPlay(true);
+        setPauseID(count);
```

Comment:
You don't want to do this inside the `setInterval` callback. I don't think it hurts having it here, but it's being called repeatedly. Both `setPlay` and `setPauseID` should be moved out.